lib/jekyll/commands/post.rb

```diff
@@ -53,6 +53,11 @@ def file_name
         def _date_stamp
           @params.date.strftime '%Y-%m-%d'
         end
+
+        def content
+          post_front_matter = Jekyll.configuration["post_front_matter"]
```

Comment:
Let's use a different configuration key here. I'd love to keep things organized under `compose`. What about:

```yaml
compose:
  post_default_front_matter:
    my: keys
    go: here
```

Top-level keys often conflict with user-specific values and it's nice to keep a plugin's keys all under one namespace.

Reply:
`jekyll_compose` is used as a top-level namespace, so I use the same naming.